internal/k8s/volume_finder.go

```diff
@@ -56,6 +72,7 @@ func (f VolumeFinder) GetPersistentVolumes() ([]VolumeInfo, error) {
 			claim := volume.Spec.ClaimRef
 			status := volume.Status
 
+			f.Logger.WithField("VolumeAttributes", volume.Spec.CSI.VolumeAttributes).Debug("volumefinder volumes attributes map")
```

Comment:
Other logging areas use snake case for field names: `volume_attributes`

Reply:
Fixed